parser/openfoodfacts_taxonomy_parser/parser/parser.py

```diff
@@ -157,27 +162,31 @@ def _create_child_links(
             RETURN COUNT(relation)
         """
 
-        language_codes = set()
+        # for unnormalised links, we need to group them by language code of the parent id
+        lc_child_links_map = {}
```

Comment:
nit: just FYI, this would be a good use case for a [collections.defaultdict](https://docs.python.org/3/library/collections.html#defaultdict-objects):
`lc_child_links_map = collections.defaultdict(list)`, no need to check for the existance of lc in map then :-)